apps/cookbook/src/app/showcase/_showcase.shared.scss

```diff
@@ -18,6 +18,7 @@ cookbook-code-viewer {
   border: 1px solid #ddd;
   border-radius: 12px;
   padding: utils.size('m');
+  margin-bottom: 24px;
```

Comment:
Are you aware that this introduces a lot of spacing in the other places the example frame is used? 😄 Is that intended? It moves the view-code button a bit far away from the example context.